content/docs/forestry/common-errors.md

```diff
@@ -0,0 +1,134 @@
+
+```yaml
+my_cool_field_foo_bar: 'hello world'
+```
+
+## Migrating Blocks
+
+The concept of using [blocks as field types in forestry](https://forestry.io/docs/settings/fields/blocks/) is supported in TinaCMS as [providing templates in an object field](/docs/editing/blocks/). During the migration, the migration tool will convert your blocks into templates in the Tina schema but it will not update your content.
+
+In TinaCMS, instead of the `template` key we use `_template` to avoid name collisions. You will have to update this in your content. As well, it is common for front matter templates to have `-` in the name and since TinaCMS does not support `-` in the name you will have to update the name of the template to use `_` instead.
+
+For example if you had a front matter template that looked like this.
+
+```yaml
+---
+label: landing-page
+fields:
+  - type: blocks
+    name: sections
+    label: Sections
+    template_types:
+      - template-name
+      - feature
+      - hero
+```
+
+And content that looked like this
+
+```yaml
+sections:
+  - template: template-name
+  #   ...
+  - template: feature
+  #   ...
+```
+
+You would have to update your content to be
+
+```yaml
+- _template: call_to_action
+#   ...
+- _template: feature
+#   ...
+```
+
+> NOTE: when you update your content you should restart your dev server to see the change in the admin
+
+Since you have updated your content you will have to update all uses of `template` in your rendering code to be `_template` and update `template-name` to be `template_name`.
+
+## Common error message and how to fix them
+
+### GraphQL error
+
+```
+Error: GraphQL Document Validation failed with 11 errors;
+  Error 0: Fields "<FieldName>" conflict because they return conflicting types "String!" and "String". Use different aliases on the fields to fetch both if this was intentional.
+```
+
+This error is because two fiends in your schema have the same name with different types and when querying for both GraphQL throws an error. This error occurs when generating the GraphQL client so if you are not using our data fetching you can mitigate this error by passing the `--noSDK` option to both `tinacms dev` and `tinacms build`.
+
+You can [read more about this issue here](https://github.com/tinacms/tinacms/issues/3150#issuecomment-1239796350)
+
+### Error when getting content in the admin
+
+```
+[Error] GetCollection failed: Unable to fetch, errors: Error querying file <File> from collection pages. Please run "tinacms audit" or add the --verbose option for more info
+```
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1673619483/tina-io/docs/forestry-migration/Screen-Shot-Error-Messager.png)
+
+This can happen for many reason but a common reason is because the data you are trying to fetch is not in the correct format, missing a required field or is missing the `_template` key. If you look at the error message in the terminal you should see the exact error that is causing the issue.
```

Comment:
```suggestion
This can happen for many reasons. Some common reasons are because the data you are trying to fetch is 
- not in the correct format 
- missing a required field
- is missing the `_template` key. 
If you look at the error message in the terminal you should see the exact error that is causing the issue.
```